Kuadrant/authorino
# Wip  Feat: Add tracing to authorino

Hello, This pr is #163 

Tasks Done 
- [x] Import open tracing packages and create a new exporter
- [x] Invoke Otel function in main() 


Verification steps courtesy @guicassolato :

```
make cluster install build

docker run --rm --name jaeger -d \
  -e COLLECTOR_ZIPKIN_HTTP_PORT=9411 \
  -p 5775:5775/udp \
  -p 6831:6831/udp \
  -p 6832:6832/udp \
  -p 5778:5778 \
  -p 16686:16686 \
  -p 14268:14268 \
  -p 9411:9411 \
  jaegertracing/all-in-one:1.6

bin/authorino server --observability-service-endpoint http://localhost:14268/api/traces
```

Then, in a separate shell:
```
curl http://localhost:5001/check

kubectl apply -f -<<EOF
apiVersion: authorino.kuadrant.io/v1beta1
kind: AuthConfig
metadata:
  name: my-authconfig
spec:
  hosts:
  - localhost
EOF

curl http://localhost:5001/check
```
Navigate to http://localhost:16686 to check the traces in the Jaeger UI.



Signed-off-by: ROHITH RAJU <rohithraju488@gmail.com>
